app/components/project-post-list.js

```diff
@@ -1,25 +1,88 @@
 import Ember from 'ember';
 
+/**
+  `project-post-list` provides a list of posts within the project and allows filtering and paging.
+
+  ## Default usage
+  ```Handlebars
+    {{project-post-list posts=posts project=project}}
+  ```
+  ## Adding post filtering
+    These additional properties allow for filtering.
+  ```Handlebars
+    {{project-post-list
+      isFiltered=isFiltered
+      isFilteringClosedPosts=isFilteringClosedPosts
+      isFilteringOpenPosts=isFilteringOpenPosts
+      filterByStatus="filterByStatus"
+      filterByType="filterByType"
+      pageNumber=page
+      posts=model
+      project=project
+      removeTypeFilter="removeTypeFilter"
+      selectedTypes=selectedTypes
+    }}
+  ```
+
+  @class Component
+  @module project-post-list
+  @extends Ember.Component
+ */
+
 export default Ember.Component.extend({
   classNames: ['project-post-list'],
 
+  /**
+    @property store
+    @type Ember.Service
+   */
   store: Ember.inject.service(),
 
+  /**
+    Returns information about the posts in order to sort and display posts in
+    the control.
+
+    @property meta
+    @type
```

Comment:
Not sure what to call this. It's pulling off of `posts`, so would it just be `DS.Model`? But at the same time `meta` isn't necessarily a defined property (I'm assuming it has something to do with meta tags in the html, but I also didn't see any Ember docs on this sort of thing.